lib/tasks/dev.rake

```diff
@@ -70,6 +71,30 @@ task({ :sample_data => :environment}) do
     p membership.errors.full_messages
   end
 
+    30.times do
+    membership = Membership.new
+    membership.group_id = Group.all[3].id
+    membership.contact_id = Contact.all.sample.id
+    membership.save
+    p membership.errors.full_messages
+  end
+
+    30.times do
+    membership = Membership.new
+    membership.group_id = Group.all[4].id
+    membership.contact_id = Contact.all.sample.id
+    membership.save
+    p membership.errors.full_messages
+  end
+
+    30.times do
+    membership = Membership.new
+    membership.group_id = Group.all[5].id
+    membership.contact_id = Contact.all.sample.id
+    membership.save
+    p membership.errors.full_messages
+  end
```

Comment:
Instead of writing this twice (or thrice), you could have another loop over the specific groups you want to ad memberships to:

Something like

```rb
  Group.all[4..5].each do |group|
    30.times do
      membership = Membership.new
      membership.group_id = group.id
      membership.contact_id = Contact.all.sample.id
      membership.save
      p membership.errors.full_messages
    end
  end
```

and then further condense it with using create and associations instead of columns

```rb
  Group.all[4..5].each do |group|
    30.times do
      membership = Membership.create(group: group, contact: Contact.all.sample)
      p membership.errors.full_messages
    end
  end
```

We can actually go even *further* by starting with the `group#memberships` association


```suggestion
  Group.all[4..5].each do |group|
    30.times do
      membership = group.memberships.create(contact: Contact.all.sample)
      p membership.errors.full_messages
    end
  end
```

how does that sound?

Reply:
amazing haha!